deploy/crd/vulnerabilityreports.crd.yaml

```diff
@@ -12,6 +12,9 @@ spec:
       storage: true
       schema:
         openAPIV3Schema:
+          description: |
+            VulnerabilityReport summarizes vulnerabilities in application dependencies and operating system packages
```

Comment:
This is just to check that kubectl explain vulnerability reports displays description. We have #409 to document the report schema.

```console
$ kubectl explain vulnerabilityreports
KIND:     VulnerabilityReport
VERSION:  aquasecurity.github.io/v1alpha1

DESCRIPTION:
     VulnerabilityReport summarizes vulnerabilities in application dependencies
     and operating system packages built into container images.


FIELDS:
   apiVersion	<string> -required-
     APIVersion defines the versioned schema of this representation of an
     object. Servers should convert recognized schemas to the latest internal
     value, and may reject unrecognized values. More info:
     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources

   kind	<string> -required-
     Kind is a string value representing the REST resource this object
     represents. Servers may infer this from the endpoint the client submits
     requests to. Cannot be updated. In CamelCase. More info:
     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds

   metadata	<Object> -required-
     Standard object's metadata. More info:
     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata

   report	<Object> -required-
```